frontend_build/src/webpack.config.base.js

```diff
@@ -44,8 +45,15 @@ aliases['keen_ui_variables'] = path.resolve('kolibri/core/assets/src/keen-config
 // helps convert to older string syntax for vue-loader
 var combineLoaders = require('webpack-combine-loaders');
 
+var postCSSLoader = {
+  loader: 'postcss-loader',
+  options: {
+    config: path.resolve(__dirname, '../../postcss.config.js')
+  }
+};
+
 // for stylus blocks in vue files
-const vueStylusLoaders = [
+var vueStylusLoaders = [
```

Comment:
I am not sure it's actually an issue in node 4, and we should probably just globally update our node to the latest anyway, as it's dev only.